### PR TITLE
Adds the option to the UI to play in off-line mode

### PR DESCRIPTION
**Pull request by Bod9001**
### Changelog:

CL: [New] Adds the option to the UI to play in off-line mode

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/GameData.cs [hunk 1]
```diff
@@ -89,6 +89,11 @@ public async void APITest()
 	}
 
 
+	public void SetForceOfflineMode(bool Value)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/GameData.cs]:
```suggestion
	public void SetForceOfflineMode(bool value)
```

File: UnityProject/Assets/Scripts/Managers/GameData.cs [hunk 2]
```diff
@@ -89,6 +89,11 @@ public async void APITest()
...
+	{
+		forceOfflineMode = Value;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/GameData.cs]:
```suggestion
		forceOfflineMode = value;
```

File: UnityProject/Assets/Scripts/UI/Systems/Lobby/AccountLoginPanel.cs [hunk 3]
```diff
@@ -26,6 +26,9 @@ public class AccountLoginPanel : MonoBehaviour
 		[SerializeField]
 		private Button exitButtonControl = default;
 
+		[SerializeField]
+		private Button OffLineButtonControl = default;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Systems/Lobby/AccountLoginPanel.cs:30]: could this be a property instead so we need no setter method like we were typing java?

**Review by corp-0 — CHANGES_REQUESTED** [review]

File: UnityProject/Assets/Scripts/UI/Systems/Lobby/AccountLoginPanel.cs (hunk 3 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/UI/Systems/Lobby/AccountLoginPanel.cs:30]: it set in the unity editor so I don't think editor likes properties

Comment (corp-0) [reply UnityProject/Assets/Scripts/UI/Systems/Lobby/AccountLoginPanel.cs:30]: wrong field, sorry

**Review by corp-0 — APPROVED** [review]

**Pull request merged**